linux/library/include/flutter_embedder.h

```diff
@@ -0,0 +1,62 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef __FLUTTER_EMBEDDER_H__
```

Comment:
We should follow Google style: https://google.github.io/styleguide/cppguide.html#The__define_Guard
So LINUX_LIBRARY_INCLUDE_FLUTTER_EMBEDDER_H_

(Also, I believe that __ as a prefix is reserved.)

Reply:
Woops!  Looks like I've been doing this wrong for a while :thinking: